src/components/NavBar.jsx

```diff
@@ -1,17 +1,31 @@
 import * as React from "react";
-import { Tab, Box } from "@mui/material";
+import { Tab, Box, Grid, AppBar, Stack } from "@mui/material";
 import { useNavigate } from "react-router-dom";
 import { Button } from "@mui/base";
+import Logo from "./Logo";
 
 export default function NavBar() {
   const navigate = useNavigate();
   return (
-    <Box sx={{ width: "100%" }}>
-      <Box sx={{ borderBottom: 1, borderColor: "divider" }}>
-        <Tab label="Home" onClick={() => navigate("/")} />
-        <Tab label="Dashboard" onClick={() => navigate("/dashboard")} />
-        <Tab label="Log in/Sign Up" onClick={() => navigate("/login-signup")} />
+    <AppBar>
+      <Box
+        sx={{ borderBottom: 1, borderColor: "divider", width: "100%" }}
+        alignItems="center"
+      >
+        <Stack direction="row">
+          <Logo />
```

Comment:
I think the Logo element needs more padding on the top, bottom and left so that the rook graphic isn't right up against the edges of the window